Add support for more arguments

davidishee opened the pull request:
I added support for arguments:

- `--ignore_dot_for_namespaces`
- `--convert_tags_to_links`
- `--tag_prop_to_taglist`
- `--journal_dashes`

As well as renaming any files using `%3A` to a `.`. I added some new test files to the test cases, but I didn't fully test all the old options. Feel free to accept the changes or not!

NishantTharani commented:
Thanks @davidishee - lots of cool changes here! Kept most of them, just deleted a snippet and updated the README + comments

davidishee commented:
Glad you liked the changes!

--
David
On Jan 4, 2024 at 2:07 PM -0600, Nishant Tharani ***@***.***>, wrote:
> Thanks @davidishee - lots of cool changes here! Kept most of them, just deleted a snippet and updated the README + comments
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you were mentioned.Message ID: ***@***.***>
